geode-core/src/main/java/org/apache/geode/internal/net/NioSslEngine.java

```diff
@@ -281,12 +281,12 @@ public ByteBufferSharing wrapNew(ByteBuffer appData)
   @NotNull
   private SSLEngineResult doWrap(final ByteBuffer appData, final ByteBuffer myNetData)
       throws SSLException {
-//    System.out.printf("BGB: pre-wrap appData: %s, myNetData: %s%n", appData, myNetData);
+    // System.out.printf("BGB: pre-wrap appData: %s, myNetData: %s%n", appData, myNetData);
```

Comment:
Commented out code should be deleted. In fact this method should probably be inlined.

Reply:
`doWrap`, `wrapOld`, `closeOld` and the `CHOOSE_BUGGY_PATH` switch will go away before this PR is finalized.

I thought it would be useful for reviewers to be able to switch the fix on/off to evaluate the tests for themselves

Reply:
fixed it